src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
```

Comment:
set transition to define properties, not to `all`, and use variable for duration

Reply:
Do you mean that you need to use `transition` on a `transition-duration`?Because I didn't quite understand what you meant.